proto/v2/v2.proto

```diff
@@ -42,5 +44,5 @@ message QueryV2Request {
 
 message QueryV2Response {
   // Keyed by the query node dcid.
-  map<string, Arc> data = 1;
+  map<string, Arcs> data = 1;
```

Comment:
I slight wonder if we can just do:

Arcs data = 1;

Because inside 'Arcs' there is also a message 'map<string, Arcs>'